src/InstrumentationEngine/CorProfilerInfoWrapper.cpp

```diff
@@ -160,13 +160,14 @@ ULONG MicrosoftInstrumentationEngine::CCorProfilerInfoWrapper::AddRef()
 
 ULONG MicrosoftInstrumentationEngine::CCorProfilerInfoWrapper::Release()
 {
-    if (InterlockedDecrement(&m_refcount) == 0)
```

Comment:
>InterlockedDecrement(&m_refcount) == 0 [](start = 8, length = 38)

what was the error here?

Reply:
I don't remember the exact error, but it was complaining that m_refcount might have changed from the time the atomic operation was performed